internal/server/http/handler_test.go

```diff
@@ -10,55 +10,61 @@ import (
 )
 
 func TestImposterHandler(t *testing.T) {
-	bodyRequest := []byte(`{
-		"data": {
-			"type": "gophers",
-		  "attributes": {
-			"name": "Zebediah",
-			"color": "Purple",
-			"age": 55
-		  }
-		}
-	  }`)
 	var headers = make(map[string]string)
 	headers["Content-Type"] = "application/json"
 
-	schemaFile := "test/testdata/imposters/schemas/create_gopher_request.json"
-	bodyFile := "test/testdata/imposters/responses/create_gopher_response.json"
-	bodyFileFake := "test/testdata/imposters/responses/create_gopher_response_fail.json"
-	body := `{"test":true}`
+	bodyXMLFile := "test/testdata/imposters/responses/create_gopher_response.xml"
+	f, _ := os.Open(bodyXMLFile)
+	defer f.Close()
+	expectedXMLBodyFileData, _ := ioutil.ReadAll(f)
```

Comment:
In theory, based on the implementation, no specific (JSON vs XML) assertion is actually required, because it works at `byte` level. But since we explicitly have support for both formats, I'd prefer to keep it explicitly on these tests as well.